src/gui/tray/usermodel.cpp

```diff
@@ -1036,6 +1036,21 @@ bool User::serverHasTalk() const
     return talkApp() != nullptr;
 }
 
+bool User::isFeaturedAppEnabled() const
+{
+    return isNcAssistantEnabled() || serverHasTalk();
+}
+
+QString User::featuredAppIcon() const
+{
+    return isNcAssistantEnabled() ? "qrc:///client/theme/white/nc-assistant-app.svg" : "qrc:///client/theme/white/talk-app.svg";
+}
```

Comment:
The recolouring you intend to do in QML won't work if you pull the images from qrc, you should use the svg image provider which can recolour the svgs